data/presets/amenity/fuel.json

```
@@ -6,12 +6,11 @@
         "operator",
         "address",
         "fuel_multi",
-        "self_service"
+        "self_service",
+        "building"
```

we should use the `building_area` field here, as quite a lot of fuel stations are mapped as points, where the building tag does not apply.

```suggestion
        "building_area"
```